.github/workflows/coverage.yml

```diff
@@ -0,0 +1,92 @@
+name: Coverage
```

Comment:
I need to think about the coverage reports. The pros / cons of a report per tool or for the entire repo.
I was using a `globcat` to merge all the lcov files together. As long as the paths are correct, this is possible.

Reply:
I'll see if turning back the `parallel` option works. I think it seemed to only be reporting the last when I did it, and I think I need to figure out the broken `flag-name` option to use it properly

Reply:
Thank you for figuring it out.